docs/utilities/worksheet_layout_utility.rst

```diff
@@ -0,0 +1,62 @@
+Utility for extending Worksheet layouts
+-----------------------------
```

Comment:
See comment above re. underlinings.
Probably you are using here a variable width font?